tensorflow/core/kernels/crop_and_resize_op.cc

```diff
@@ -250,22 +251,21 @@ struct CropAndResize<CPUDevice, T> {
             continue;
           }
           if (method_name == "bilinear") {
-            const int top_y_index = floorf(in_y);
-            const int bottom_y_index = ceilf(in_y);
+            const int bottom_y_index = std::min(top_y_index + 1, image_height - 1);
```

Comment:
This will not work correctly if in_y is an integer, i.e., `in_y == top_y_index == bottom_y_index`.

Reply:
@penpornk, Thanks for review! Al this PR is driven by specific test case (see changed file). As far as I remember, without the changes in CropAndResize op it had last line with zeros.

Reply:
Modified it. Let's check the test.

Reply:
Thanks for the clarification! 
I reran the tests. 9 of them failed.